### PR TITLE
tweaks

### PR DESCRIPTION
**Pull request by SamVerschueren**
Added some tweaks
- Documentation
- Added [EditorConfig](https://editorconfig.org) - Must have in teams to keep consistency :)
- Fixed some things in `package.json`

Please review, if you find things that you want differently or that you rather see reverted back to what it was, let me know.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```
Comment (SamVerschueren) [inline .editorconfig]: Not sure on this one, do you use 4 or 2?

Comment (SamVerschueren) [reply .editorconfig]: It looks like you are using 4. I will fix it.

File: package.json [hunk 2]
```diff
@@ -16,18 +27,8 @@
     "buildprod": "npm run buildprod:js && npm run buildprod:umd && npm run buildprod:uglify",
     "deploy": "npm run buildprod && cd ./release && npm publish"
   },
-  "bugs": {
-    "url": "https://github.com/brechtbilliet/ngrx-undo/issues"
-  },
-  "homepage": "https://github.com/brechtbilliet/ngrx-undo#readme",
```
Comment (SamVerschueren) [inline package.json:22]: `bugs` and `homepage` are automatically derived from `repository`.

**Comment by SamVerschueren** [comment]
I just noticed I could've done another tweak in the concrete example.

``` js
remove(wine: Wine): void {
```

Is that valid? Shouldn't it be `function remove(wine: Wine) {` instead?

**Comment by brechtbilliet** [comment]
It's a typescript class, not javascript. But I get your point, it might not be clear that it's a class

**Pull request merged**